packages/utils/src/validators.ts

```diff
@@ -286,3 +286,10 @@ export function isValidNamespacesEvent(
   const events = getNamespacesEventsForChainId(namespaces, chainId);
   return events.includes(eventName);
 }
+
+export function isValidNamespacesChange(
+  requiredNamespaces: ProposalTypes.RequiredNamespaces,
+  namespaces: SessionTypes.Namespaces,
+) {
+  return !Object.keys(getDeletedDiff(requiredNamespaces, namespaces)).length;
```

Comment:
I think the simpler semantics would be to avoid the negation and explicitly do `Object.keys(...).length === 0`